stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGallery.kt

```diff
@@ -63,8 +65,19 @@ public class AttachmentGallery : ConstraintLayout {
         configureImagesMenu()
     }
 
-    public fun provideImageList(fragmentActivity: FragmentActivity, imageList: List<String>, currentIndex: Int = 0) {
-        adapter = AttachmentSlidePagerAdapter(fragmentActivity, imageList)
+    public fun provideImageList(
+        fragmentActivity: FragmentActivity,
+        imageList: List<String>,
+        currentIndex: Int = 0,
+        imageClickListener: () -> Unit = {},
+    ) {
+        adapter = AttachmentSlidePagerAdapter(fragmentActivity, imageList) {
+            binding.bottomBarGroup.isVisible = isFullScreen
+
+            isFullScreen = !isFullScreen
+
+            imageClickListener()
+        }
```

Comment:
The last lambda parameter is not very obvious here at the call site, it's hard to tell when it will be invoked. Being more verbose would be useful here, with something like:

```kotlin
AttachmentSlidePagerAdapter(
    fragmentActivity,
    imageList,
    imageClickListener = {
       binding.bottomBarGroup.isVisible = isFullScreen
       isFullScreen = !isFullScreen
       imageClickListener()
    }
)
```